MTAOS/v4/_summit.yaml

```diff
@@ -4,7 +4,7 @@ data_path: /repo/LSSTComCam
 run_name: mtaos_wep
 collections: "LSSTComCamSim/raw/all,LSSTComCamSim/calib,refcats"
 pipeline_n_processes: 9
-zernike_table_name: zernikeEstimateAvg
+zernike_table_name: zernikeEstimateRaw
```

Comment:
you want to go back to `zernikeEstimateRaw`? I thought the idea was to use `zernikeEstimateAvg`.